project1/templates/bookpage.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Book page</title>
+  
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css">
+  <link rel="shortcut icon" href="{{ url_for('static', filename='homework.png') }}">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"></script>
+</head>
+<body background="{{ url_for('static', filename='bimage.jpg') }}">
+
+ 
+<div class="container">
+  <div class="jumbotron">
+  <h2 style = "color: blue;text-align: center;"> Book page</h2>
+  <p style = "color: black;text-align: center;">Below are the book details</p>
+  <nav class="navbar navbar-dark bg-dark justify-content-between">
+    <a class="navbar-brand">{{ user }}</a>
+    <form class="form-inline">
+        <button class="btn btn-outline-primary my-2 my-sm-0" type="submit" formaction = '/logout'>Logout</button>
+    </form>
+  </nav>
```

Comment:
This code should be at the start of the body so that Navbar will be displayed on the top